src/views/layout.njk

```diff
@@ -18,7 +18,7 @@
           crossorigin="anonymous"></script>
 
   <script type="application/javascript">
-    window.SERVICE_NAME = 'appeals'
+    window.SERVICE_NAME = '{{ serviceName }}'
```

Comment:
Where is {{ serviceName }} going to pick its value from?

Reply:
https://github.com/companieshouse/lfp-appeals-frontend/blob/4965d4671d36acc4fb7942062552c8e7dd5068ad/src/views/layout.njk#L8